exporter/tile38_test.go

```diff
@@ -33,12 +34,16 @@ func TestTile38(t *testing.T) {
 			close(chM)
 		}()
 
+		metricPattern := regexp.MustCompile(`^(tile38){1}(_[a-z0-9]+)+$`)
 		found := false
 		want := "tile38_threads_total"
 		for m := range chM {
-			if strings.Contains(m.Desc().String(), want) {
+			if strings.EqualFold(m.Desc().String(), want) {
```

Comment:
Equal won't work as the metrics are prefixed with whatever prefix the exporter has set, in case of the test exporter that's `test_`.
You can see the tests failing here: https://cloud.drone.io/oliver006/redis_exporter/1137/1/16

I'm not too worried about unicode and whatnot and would just stick to Contains() but I think adding checks for a few more metrics would be good to make sure you're exporting them correctly.